build/common.sh

```diff
@@ -55,7 +43,16 @@ readonly KUBE_BUILD_PPC64LE="${KUBE_BUILD_PPC64LE:-n}"
 # Constants
 readonly KUBE_BUILD_IMAGE_REPO=kube-build
 readonly KUBE_BUILD_IMAGE_CROSS_TAG="$(cat ${KUBE_ROOT}/build/build-image/cross/VERSION)"
-# KUBE_BUILD_DATA_CONTAINER_NAME=kube-build-data-<hash>"
+# KUBE_DATA_CONTAINER_NAME=kube-build-data-<hash>"
+
+# This version number is used to cause everyone to rebuild their data containers
+# and build image.  This is especially useful for automated build systems like
+# Jenkins.
+#
+# Increment/change this number if you change the build image (anything
+# under build/build-image, golang version) or change the set of volumes in the
+# data container.
+readonly KUBE_BUILD_IMAGE_VERSION=1
```

Comment:
Hmmm -- that is interesting.  Right now updating `CROSS_TAG` won't cause a rebuild.  But before it was half broken as a `CROSS_TAG` update wouldn't cause data container to be blown away.

I'll update it to be a separate file and mix in CROSS_TAG.